### PR TITLE
 lines_cop: add ENV.universal_binary audit exemption for wine

### PR DESCRIPTION
**Pull request by JCount**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [x] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/homebrew/pull/49031).
- [x] Have you successfully run `brew tests` with your changes locally?

-----
add ENV.universal_binary audit exception for wine and corresponding test coverage

### CODE REVIEW

**Comment by JCount** [comment]
cc @ilovezfs

File: Library/Homebrew/test/rubocops/lines_cop_spec.rb [hunk 1]
```diff
@@ -419,7 +419,22 @@ class Foo < Formula
       end
     end
 
-    it "with ENV.universal_binary" do
+    it "with ENV.universal_binary exempted formula" do
```
Comment (reitermarkus) [inline Library/Homebrew/test/rubocops/lines_cop_spec.rb]: Can you make this into a proper sentence? Thanks. 😉

Comment (JCount) [reply Library/Homebrew/test/rubocops/lines_cop_spec.rb]: I was only matching the one above it. So, inconsistency or proper sentences?

Comment (reitermarkus) [reply Library/Homebrew/test/rubocops/lines_cop_spec.rb]: Ideally proper sentences for all, or `specify` instead of `it`.

Comment (JCount) [reply Library/Homebrew/test/rubocops/lines_cop_spec.rb]: The full fragment is: `' When auditing formula with ENV.universal_binary exempted formula'` any suggestions?

Comment (JCount) [reply Library/Homebrew/test/rubocops/lines_cop_spec.rb]: (I am referring to the line https://github.com/Homebrew/brew/pull/3291/files#diff-9349b511bf2b9967d30735fbb26db7b7R383)

**Comment by JCount** [comment]
@reitermarkus Is something like this what you had in mind?

**Comment by JCount** [comment]
@reitermarkus gentle ping on this

**Comment by reitermarkus** [comment]
[Something like this](https://github.com/Homebrew/brew/pull/3294#discussion_r144666636) is what I had in mind.

**Comment by JCount** [comment]
Since this has the context `When auditing formulae` prepended to both tests, making the wording following the `it` into complete sentences, would, in my mind, only serve to make things less intelligible.

**Comment by ilovezfs** [comment]
`it "is as quiet as a church mouse when there is an exemption to the deprecation of ENV.universal_binary" do`

**Comment by reitermarkus** [comment]
> …  in my mind, only serve to make things less intelligible.

It should be possible to make it more intelligible than “when auditing formulae it with a build.universal? exemption”.

Also, the context is pretty redundant, so this could actually be removed, and we'd just have `it "does not report build.universal? offenses for Wine" do`, which would be pretty clear to me.

**Pull request merged**